src/main/java/com/easypost/model/TimeInTransitDetailsForDeliveryDate.java

```diff
@@ -0,0 +1,12 @@
+package com.easypost.model;
+
+import java.util.Date;
+import lombok.Getter;
+
+@Getter
+public class TimeInTransitDetailsForDeliveryDate {
```

Comment:
I believe you got this class confused, this should have the following fields:

"planned_ship_date", "easypost_estimated_delivery_date", "days_in_transit"

You then need to have a TimeInTransitDetailsForShipDate, with the following fields:

"desired_delivery_date", "ship_on_date", "delivery_date_confidence", "estimated_transit_days", "days_in_transit"

Reply:
Heh, the `results` for both endpoints differ. I have created `TimeInTransitDetailsForShipDate` for the `deliver_on` endpoint. Good catch! `planned_ship_date` should not be included in this class, I didn't see it in the API response.

![Screenshot 2024-07-12 at 4 56 05 PM](https://github.com/user-attachments/assets/0592ebf8-d7ec-4f09-97d8-4d3246da283e)